mcs/class/referencesource/System.Web.DataVisualization/Common/ChartTypes/BoxPlotChart.cs

```diff
@@ -1,6 +1,6 @@
 //-------------------------------------------------------------
-// <copyright company=�Microsoft Corporation�>
-//   Copyright � Microsoft Corporation. All Rights Reserved.
+// <copyright company=�Microsoft Corporation�>
+//   Copyright � Microsoft Corporation. All Rights Reserved.
```

Comment:
Hm. Looks like these files are in the Windows-1252 encoding and your changes are in some other encoding.  Could you convert them to utf-8 instead?

Reply:
How about just sticking to 7 bit ASCII?

Reply:
I can't because I'm using either notepad ++ or VS Code.

Reply:
![image](https://user-images.githubusercontent.com/1798510/73120328-c1489a00-3f6c-11ea-800e-cde12347d8d3.png)

Reply:
@lambdageek I made sure it is UTF-8. I replaced those weird characters with their previous values in order to fix it.